### ISSUE TITLE
rien ne sort des commandes 

### ISSUE THREAD

**Issue by ralessi** [maieul/ledmac#332]

**Comment by ralessi** [comment]
Gros problème, et là encore, je ne vois pas comment faire de mwe.

Avec petit un fichier de test, les commandes de type:
````
\psart
\eledchapter*{Essai de \edtext{titre}{\Bfootnote{bla bla}}}
\pend
````
fonctionnent parfaitement.

Mais dans mon édition, la même commande sort le titre, mais ne met rien dans les notes critiques. Même une commande de type `\ledrightnote` ne sort rien dans la marge.

Voici un extrait:
````
\begin{Rightside}
\linenummargin{inner}
\lineation{page}

\beginnumbering
%%hunayn-begin
% \setlinenum{1}
% \firstlinenum{5}\linenumincrement{5}
%%hunayn-end

\pstart
\eledchapter*{Ἰπποκράτους ἐπιδημιῶν \edtext{βιβλίον
    δεύτερον}{\Bfootnote{IRH: λόγος β᾿ V}}\ledrightnote{Oup}}
\pend
[...]
````
Aucune erreur de compilation pourtant.

**Comment by maieul** [comment]
malheureusement sans MWE je ne peux rien faire� que donne les fichier .eledsec?

**Comment by ralessi** [comment]
Je les avais regardés, et tout me semblait bon. Voici un exemple:
````
\eled@chapter{}{Ἰπποκράτους ἐπιδημιῶν \edtext {βιβλίον δεύτερον}{\Bfootnote {IRH: λόγος β᾿ V}}}{1}{*}{R} 
\eled@section{}{<\edtext {Τμῆμα πρῶτον}{\Bfootnote {Littré: om. codd.}}>}{2}{*}{R} 
````

**Comment by ralessi** [comment]
Ah quand même une chose: une commande de type `\ledsidenote{}` dans un `\eledchapter` ne sort rien.

Pas besoin de mwe, puisque c'est le cas du fichier exemple dans la doc actuelle d'eledmac.

**Comment by ralessi** [comment]
Je veux parler de `2-titles_in_line_numbering_with_notes.tex`.

Pour les notes critiques elles-mêmes, elles passent dans un petit fichier simple. Mais pas quand le fichier devient complexe comme c'est le cas.

J'ai déjà été confronté à un cas de disparition «silencieuse» de lignes de notes critiques, c'était dans la issue #239 avec `\numberpstarttrue` (ticket fermé pour une autre raison).

**Comment by ralessi** [comment]
Pardon: #329

**Comment by ralessi** [comment]
#329 premier post.

**Comment by maieul** [comment]
indeed, il y a pb sur le fichier en question : les num�ros de lignes n'apparaissent plus. Mais lorsque j'ai sorti \eledchapter, cela marchait.

Donc je suppose que le bug est venu entre temps. Un git bisect devrait me permettre de trouver rapidement la cause

**Comment by ralessi** [comment]
> indeed, il y a pb sur le fichier en question : les numéros de lignes n'apparaissent plus. Mais lorsque j'ai sorti \eledchapter, cela marchait.
> 
> Donc je suppose que le bug est venu entre temps. Un git bisect devrait me permettre de trouver rapidement la cause

Ah, et moi qui ai reformaté mes titres! Je pensais que c'était trop compliqué. En fait, ce problème m'a donné l'occasion de regarder de plus près le formatage qui est souhaité pour mon édition: 
- faire démarrer les numéros de ligne *après* le titre, ce qui n'est pas compliqué;
- pour le titre lui-même: y renvoyer dans l'apparat par «Tit.», et supprimer la numérotation.

Mais tout compte fait, avec des commandes \eledchapter{} qui marchent on peut arriver à ce résultat facilement.

**Comment by maieul** [comment]
bon, trouv� la cause. C'est vraiment du TeX subtile (4d7dbef). Peux tu tester la branche issue332 ? cela r�soud le pb du fichier 2-xxx, mais je ne suis pas s�r que cela r�solve ton autre pb.

**Comment by ralessi** [comment]
> bon, trouv� la cause. C'est vraiment du TeX subtile (4d7dbef). Peux tu tester la branche issue332 ? cela r�soud le pb du fichier 2-xxx, mais je ne suis pas s�r que cela r�solve ton autre pb.

Malheureusement, je n'ai toujours pas les notes critiques qui correspondent au titre. J'ai mis ceci:
````
\pstart
\eledchapter*{Ἰπποκράτους ἐπιδημιῶν \edtext{βιβλίον
    δεύτερον}{\Bfootnote{IRH: λόγος β᾿ V}}}
\pend
````

Mais je n'ai pas ces variantes dans les notes qui commencent à la première ligne de texte.

Autre remarque: j'ai dû compiler 6 fois après ce simple changement pour avoir un nombre de page stable.

**Comment by ralessi** [comment]
Attends!

Je me suis trompé de numéro: 322 au lieu de 332.

**Comment by maieul** [comment]
a oui, issue322 n'aurait pas du rester�
je supprime la branche

**Comment by ralessi** [comment]
Hélas les notes ne sont pas passées. Par ailleurs, le style de titre devrait produire un titre centré, et ce n'est pas le cas.

Voici ce que cela donne pour le titre:
![capture](https://cloud.githubusercontent.com/assets/389498/8544265/3296751c-24a4-11e5-9159-95720675cc41.png)

Et pour le début des notes:
![capture-1](https://cloud.githubusercontent.com/assets/389498/8544277/4f763140-24a4-11e5-80e4-c60793aaa019.png)

**Comment by maieul** [comment]
il est pr�cis� que le \eledsection ne fonctionnent qu'avec les styles standards. Je paris que tu utilise titlesec. Ceci expliquerait que tes notes aient disparus.

**Comment by ralessi** [comment]
Tu viens de gagner ton pari. Délosé de t'avoir fait perdre du temps.

Mais où cela est-il précisé dans la documentation? Je viens de relire le point 14 p. 48-49 et je n'ai pas vu.

Cela veut-il dire que l'on ne peut pas personnaliser les styles de chapitre par rapport à celui que fournit la classe book?

**Comment by maieul** [comment]
> Tu viens de gagner ton pari. D�los� de t'avoir fait perdre du temps.

pas une perte de temps, j'ai vu un autre bug entre temps. Je vais sortir une version pour ce bug.
> 
> Mais o� cela est-il pr�cis� dans la documentation? Je viens de relire le point 14 p. 48-49 et je n'ai pas vu.

je crois que je confond. je l'avais document� pour \ledchapter, mais j'ai du oubli� pour \eledchapter
> 
> Cela veut-il dire que l'on ne peut pas personnaliser les styles de chapitre par rapport � celui que fournit la classe book?

pour le moment oui. Il faudrait que je me penche sur le fonctionnement interne de titlesec pour voir comment on peut faire. Il s'agit "juste" de patcher correctement les commandes. Le probl�me est qu'il n'est pas forc�ment simple de savoir comment patcher.

Donc : en suspens, en attendant la sortie de reledmac et la r�solution des autres bugs

**Comment by ralessi** [comment]
Je suis rassuré.

Oui, il faut laisser en suspens. Je me demande même s'il est souhaitable d'entrer dans le fonctionnement interne d'autres packages, car risque d'introduire une dépendance. Book, c'est bien.

Certes, le formatage des titres est une chose importante. Mais on peut aussi s'en tirer à la main, quitte à ajouter un `\addcontentsline{}{}{}` pour avoir le titre dans la table...

**Comment by maieul** [comment]
Le 7 juil. 2015 � 12:58, Robert Alessi <notifications@github.com> a �crit :

> Je suis rassur�.
> 
> Oui, il faut laisser en suspens. Je me demande m�me s'il est souhaitable d'entrer dans le fonctionnement interne d'autres packages, car risque d'introduire une d�pendance. Book, c'est bien.
> 

titlesec est quand m�me utile. ce n'est pas comme memoir: il ne fait qu'une chose. Du coup on peut rentrer sans trop de risque dedans, surtout avec les \patchcmd.

**Comment by Doc73** [comment]
@maieul 
I too came across this bug. Have you time to give a look at this issue?

I could provide a simple MWE, with `reledpar`:
```
% !TeX program = xelatex
% !TeX encoding = UTF-8
% !TeX spellcheck = it_IT

\documentclass[11pt,a4paper]{book}
\usepackage{libertine}

\usepackage{titlesec}
\titleformat{\chapter}
	{\centering}%format
	{}%label
	{00pt}%sep
	{\large\MakeUppercase}%before
\titleformat{\section}
	{\centering}%format
	{}
	{00pt}
	{}

\usepackage{reledmac,reledpar}


\begin{document}

\begin{pages}
\begin{Leftside}
\beginnumbering
\pstart\eledchapter{Chapter\ledsidenote{bla}}\pend

\pstart
\eledsection{\edtext{Θεωρία}{\lemma{Osservazione preliminare}\Aendnote{nota finale}}\ledsidenote{>}}
\pend

\pstart Nel mezzo del cammin di nostra vita\pend
\endnumbering
\end{Leftside}

\begin{Rightside}
\beginnumbering
\pstart\eledchapter{Chapter}\pend
\pstart\eledsection{Section}\pend
\pstart Nel mezzo del cammin di nostra vita\pend
\endnumbering
\end{Rightside}
\end{pages}
\Pages

\doendnotes{A}
\end{document}
```

**Comment by maieul** [comment]
sorry, now, I have no time.
